### DISCUSSION TITLE
I'm trying to mixin to the head of the scrollInHotbar method in the PlayerInventory class in order to prevent scrolling in the hotbar. But when I run the game, I'm still able to scroll in the hotbar. Please help me out

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#2808] [Mod Dev Support] [answered]
https://gist.github.com/TechPro424/debaa7b592b695dd33c4633606b42e8d
I'm trying to mixin to the head of the scrollInHotbar method in the PlayerInventory class in order to prevent scrolling in the hotbar. But when I run the game, I'm still able to scroll in the hotbar. Please help me out

**Comment by warjort** [comment, marked as answer]
Returning normally from a mixin just lets the original code continue.

https://fabricmc.net/wiki/tutorial:mixin_examples#injecting_and_cancelling

**Reply by TechPro424** [reply]
That worked, thanks for your help! And Happy New Year!
